btk/draw_blends.py

```diff
@@ -355,9 +359,11 @@ def render_mini_batch(self, blend_list, psf, wcs, survey, noise_seed, extra_data
                 (self.max_number, len(survey.filters), pix_stamp_size, pix_stamp_size)
             )
             blend_image_multi = np.zeros((len(survey.filters), pix_stamp_size, pix_stamp_size))
+            seeds_blend = seed_minibatch.spawn(len(survey.filters))
             for b, filt in enumerate(survey.filters):
+                seed_blend = seeds_blend[b]
                 single_band_output = self.render_blend(
-                    blend, psf[b], filt, survey, noise_seed, extra_data[i]
+                    blend, psf[b], filt, survey, seed_blend, extra_data[i]
```

Comment:
all these are seed sequences and not seed values (integers), again this is not made clear enough IMO.